### DISCUSSION TITLE
Error while compiling in angular 6

### DISCUSSION THREAD

**Discussion by nithin1204** [fingerprintjs/fingerprintjs#656] [Q&A] [answered]
I had included fingerprintjs using **npm i @fingerprintjs/fingerprintjs** in my angular application. While compiling im getting the error. My current typescript version in 2.9.2

ERROR in ../node_modules/@fingerprintjs/fingerprintjs/dist/fp.d.ts(281,197): error TS1005: ',' expected.
../node_modules/@fingerprintjs/fingerprintjs/dist/fp.d.ts(281,205): error TS1005: ',' expected.

**Comment by Finesse** [comment, marked as answer]
Please check the [TypeScript support guide](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/typescript_support.md)

**Reply by nithin1204** [reply]
Many thanks @Finesse :)
